ecosystem/typescript/sdk_v2/src/utils/hexData.ts

```diff
@@ -0,0 +1,67 @@
+import { bytesToHex, hexToBytes } from "@noble/hashes/utils";
+import { Hex } from "../types";
+
+export class HexData {
+  private hexData: Hex;
+
+  /**
+   * Creates new HexData instance from Hex type.
+   * @param hex Uint8Array
+   */
+  constructor(hex: Hex) {
```

Comment:
```suggestion
  constructor(hex: Uint8Array) {
```

I assume you meant this, right? Given the `@param hex Uint8Array` notation and the usage of `fromString`